src/locale/en.json

```
@@ -375,14 +370,14 @@
   "terug_naar_regio_selectie": {
     "text": "Back to selection of region"
   },
+  "over_risiconiveas": {
```

Missing an "u" here.

```suggestion
  "over_risiconiveaus": {
```